mne_bids/tests/test_write.py

```diff
@@ -639,4 +644,59 @@ def test_write_anat(_bids_validate):
     # specify trans but not raw
     with pytest.raises(ValueError, match='must be specified if `trans`'):
         write_anat(output_path, subject_id, t1w_mgh, session_id, raw=None,
-                   trans=trans, verbose=True, overwrite=True)
+                   trans=trans, verbose=True, deface=False, overwrite=True)
+
+    # test deface
+    if not has_nibabel():  # pragma: no cover
+        raise ImportError('This function requires nibabel.')
```

Comment:
don't do this like this. Use a decorator so that test is just skipped if nibabel is not installed. Now I don't think the whole test should be skipped with nibabel is not available.